options-and-settings-pages/genesis-cpt-archive-metabox.php

```diff
@@ -92,30 +101,36 @@ public function hooks() {
 
 	/**
 	 * Initiate admin hooks.
-	 * @since  0.1.0
+	 *
+	 * @since 0.1.0
 	 */
 	public function init() {
-		// Add custom archive support for CPT
+		// Add custom archive support for CPT.
 		add_post_type_support( $this->post_type, 'genesis-cpt-archives-settings' );
 	}
 
 	/**
 	 * Add admin hooks.
+	 *
 	 * @since 0.1.0
 	 */
 	public function admin_hooks() {
-		// Include CMB CSS in the head to avoid FOUC
+		// Include CMB CSS in the head to avoid FOUC.
 		add_action( "admin_print_styles-{$this->admin_hook}", array( 'CMB2_hookup', 'enqueue_cmb_css' ) );
 
 		// Hook into the genesis cpt setttings save and add in the CMB2 sanitized values.
-		add_filter( "sanitize_option_genesis-cpt-archive-settings-{$this->post_type}", array( $this, 'add_sanitized_values' ), 999 );
+		add_filter( "sanitize_option_genesis-cpt-archive-settings-{$this->post_type}", array(
+			$this,
+			'add_sanitized_values',
+		), 999 );
```

Comment:
not sure how the sniffer missed this, but callback arrays should not be split to multiple lines.